sdk/servicebus/azure-servicebus/azure/servicebus/aio/async_client.py

```diff
@@ -36,6 +36,9 @@ class ServiceBusClient(mixins.ServiceBusMixin):
     :param str host_base: Optional. Live host base URL. Defaults to Azure URL.
     :param str shared_access_key_name: SAS authentication key name.
     :param str shared_access_key_value: SAS authentication key value.
+    :param transport_type: Optional. Underlying transport protocol type (Amqp or AmqpOverWebsocket)
+     Default value is ~uamqp.constants.TransportType.Amqp
+    :type transport_type: ~uamqp.constants.TransportType
```

Comment:
I would prefer hide the concept of uamqp, as now we've exposed TranportType at the azure.servicebus module level.
```
    :param transport_type: Optional. Underlying transport protocol type (Amqp or AmqpOverWebsocket)
     Default value is ~azure.servicebus.TransportType.Amqp
    :type transport_type: ~azure.servicebus.TransportType
```